2/contrib/openshift/base-plugins.txt

```diff
@@ -4,7 +4,7 @@ openshift-client:1.0.5
 
 
 # kubernetes plugin - https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin
-kubernetes:1.1.3
+kubernetes:1.2
```

Comment:
OK, if we are going to bump this, we need to do a sanity check and make sure our basic k8s pods as slaves tests still pass.

I'll check out this branch, build the image, and run a regression.  We still don't have the infrastructure to do this directly from the PR (but as @bparees knows, I'd like to get a card on the books to sort that out ;-) ).

Reply:
my regression test using the image from this PR passed.